src/lib.rs

```diff
@@ -781,12 +627,10 @@ bsize: BlockSize, bo: &BlockOffset) -> f64 {
     }
 
     let bs = mi_size_wide[bsize as usize];
-    let is_too_big =
-        bo.x as i32 + bs as i32 - fi.w_in_b as i32 > 0 ||
-        bo.y as i32 + bs as i32 - fi.h_in_b as i32 > 0;
+    let must_split = bo.x + bs as usize > fi.w_in_b || bo.y + bs as usize > fi.h_in_b;
 
     let is_splittable = bsize > fi.min_partition_size;
-    let is_codable = bsize < BlockSize::BLOCK_64X64 && !is_too_big;
+    let is_codable_unsplit = bsize < BlockSize::BLOCK_64X64 && !must_split;
```

Comment:
I think the term "bsize < BlockSize::BLOCK_64X64" is required only because tx64x64 is not working now.
(Which, I know from my wip patch for enabling tx64x64)
So, if we add that term to line 630, like
let must_split = bo.x + bs as usize > fi.w_in_b || bo.y + bs as usize > fi.h_in_b || bsize > BlockSize::BLOCK_32X32
then
line 633 simply becomes :  let is_codable_unsplit = !must_split;
In that sense, what tdaede inquired can be true. I.e if it is not 'must split' (for whatever reason) then it 'is_codable' == true.
So, for two reasons, 1) future integration with "enabling tx64x64" and 
2) "bsize < BlockSize::BLOCK_64X64" is better placed in must_split = ...
i.e. if no_split (i.e. must_split == false), then always is_codable == true.
I'd suggest:
must_split = bo.x + bs as usize > fi.w_in_b || bo.y + bs as usize > fi.h_in_b || bsize > BlockSize::BLOCK_32X32;
is_codable == !must_split;

Reply:
This was already addressed in the last commit.